### PR TITLE
Initial code commit.

### PR DESCRIPTION
**Pull request by j0hnmwangi**

### CODE REVIEW

**Review by patrickmukumbu — CHANGES_REQUESTED** [review]
See my comment @JohnNMwangi

**Comment by j0hnmwangi** [comment]
> See my comment @JohnNMwangi

Hey @patrickmukumbu  the comments aren't visible.

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,77 @@
+
+variable "description" {
+  default     = ""
+  description = "The description of the key as viewed in AWS console."
+  type        = string
+}
+
```
Comment (patrickmukumbu) [inline variables.tf]: For the variables with default being `" "` are they required? If not so, can they be defaulted to `null`

**Review by patrickmukumbu — CHANGES_REQUESTED** [review]

**Review by patrickmukumbu — APPROVED** [review]
LGTM! Everything works fine on testing

**Pull request merged**